client/blocks/post-item/index.jsx

```diff
@@ -210,7 +222,7 @@ class PostItem extends React.Component {
 						globalId={ globalId }
 						onClick={ this.clickHandler( 'image' ) }
 					/>
-					{ ! multiSelectEnabled && <PostActionsEllipsisMenu globalId={ globalId } /> }
+					{ this.renderEllipsisMenu() }
```

Comment:
As simple as this function is could we consider inlining it here for the sake of more clearly seeing what's rendered?

Reply:
While this probably wouldn't affect performance in a noticeable way, I think it's good to avoid creating functions in each render call.

Reply:
Ah, on second thought we'd avoid the function creation in that case with a ternary I suppose? imo it's a toss up for readability, so whatever you prefer @Copons